src/translations/strings.yaml

```diff
@@ -928,6 +928,19 @@ devices:
     value: My devices
     comment: Title for the menu bar when viewing the devices or device limit pages
 
+  privacyTitle:
+    value: Privacy features
+    comment: Title label for the privacy features help sheet
+  privacyHeader:
+    value: What do privacy features do for me?
+    comment: Header label for the privacy features help sheet
+  privacyBody1:
+    value: Privacy features give you greater control over the types of content you see, helping you stay safe online.
+    comment: Body label for the privacy features help sheet
+  privacyBody2:
+    value: If you activate these features, they’ll overwrite any custom DNS you’re using. These protections are not a substitute for taking other security precautions — for example, while blocking malware you should still avoid downloading attachments in strange emails.
+    comment: Body label for the privacy features help sheet
```

Comment:
Nit: Do we need to break up the body into two strings? Is it more flexible for localization if a single string (and a single Text field in the UI) is used? 

Feel free to ignore this comment.

Reply:
In this case, one string provides more flexibility for translation.

Reply:
For layout/spacing reasons, we have to use two text components here